packages/rke2-kube-proxy/charts/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: rke2-kube-proxy
 description: Install Kube Proxy.
-version: v1.21.1-build20210520
-appVersion: v1.21.1
+version: v1.20.7-build20210520
+appVersion: v1.20.7
```

Comment:
Wondering how we control in rke2-charts the release. Will this generate an extra release for the kube-proxy chart that we can consume with version v1.20.7-build20210520? Or will it remove the v1.21.1-build20210520?

Reply:
The package version gets appended to this, so it will create a new chart version: v1.20.7-build2021052005